state/keys.go

```diff
@@ -14,6 +18,8 @@ const (
 	All              = Read | Allocate | Write
 )
 
+var ErrMalformedEncoding = errors.New("malformed encoding")
```

Comment:
not sure if this is the right place.. but please lmk if you think elsewhere would be a better place.